src/scrobbler/listenbrainzscrobbler.cpp

```diff
@@ -447,6 +448,19 @@ void ListenBrainzScrobbler::UpdateNowPlaying(const Song &song) {
 
   object_track_metadata.insert("track_name", QJsonValue::fromVariant(title));
 
+  QJsonObject object_additional_info;
+
+  object_additional_info.insert("duration_ms", song.length_nanosec() / kNsecPerMsec);
+
+  if (const int track = song.track(); track > 0) {
+    object_additional_info.insert("tracknumber", track);
+  }
+
+  object_additional_info.insert("submission_client", QCoreApplication::applicationName());
+  object_additional_info.insert("submission_client_version", QCoreApplication::applicationVersion());
```

Comment:
ListenBrainz also has `media_player` and `media_player_version` fields, since they are the same thing I didn't include them. I'm not sure which one of them has higher priority, I'll ask about that on their IRC now.

Reply:
I asked on IRC, both should be sent. Example: https://listenbrainz.readthedocs.io/en/latest/users/json.html#brainzplayer-on-the-listenbrainz-website-playing-a-video-from-youtube.